README.md

```diff
@@ -215,12 +215,80 @@ Body: {
 
 ### Job Applications
 
+#### Fetch all Job Applications for a user
+Request:
+```
+GET /api/v1/users/:user_id/job_applications
+```
+Headers:
+```
+{
+  "Authorization": "Bearer <your_token_here>"
+}
+```
+Successful Response:
+```
+
+Status: 200
+
+{
+    "data": [
+        {
+            "id": "1",
+            "type": "job_application",
+            "attributes": {
+                "position_title": "Jr. CTO",
+                "date_applied": "2024-10-31",
+                "status": 1,
+                "notes": "Fingers crossed!",
+                "job_description": "Looking for Turing grad/jr dev to be CTO",
+                "application_url": "www.example.com",
+                "contact_information": "boss@example.com",
+                "company_id": 1
+            }
+        },
+        {
+            "id": "3",
+            "type": "job_application",
+            "attributes": {
+                "position_title": "Backend Developer",
+                "date_applied": "2024-08-20",
+                "status": 2,
+                "notes": "Had a technical interview, awaiting decision.",
+                "job_description": "Developing RESTful APIs and optimizing server performance.",
+                "application_url": "https://creativesolutions.com/careers/backend-developer",
+                "contact_information": "techlead@creativesolutions.com",
+                "company_id": 3
+            }
+        }
+    ]
+}
+```
+Error Response if no token provided:
+
+```
+Status: 401 Unauthorized
+
+Body: {
+    "message": "Invalid login credentials",
+    "status": 401
+}
+```
+
 #### Create a Job Application
 
 Request:
 ```
 POST /api/v1/users/:user_id/job_applications
-
+```
+Headers:
+```
+{
+  "Authorization": "Bearer <your_token_here>"
+}
```

Comment:
When I was playing around with this in Postman days ago, one initially had to find the token of a seeded user on the db, then paste that token into the header with Key: "Authorization" Value: <encoded token>.

Note that I did not include "Bearer" as well as ensuring that the encoded token is not set as a string.  Is this still accurate?

Reply:
The app currently doesn't require bearer, but it will accept a token with or without bearer. I added it in there because it's good common practice as many APIs expect Bearer, and it follows OAuth 2.0 conventions.

Reply:
Good to know, thanks.